internal/operators/manager_test.go

```diff
@@ -821,10 +821,10 @@ var _ = Describe("Operators manager", func() {
 
 		It("ListBundle should return the list of available bundles", func() {
 			bundles := manager.ListBundles()
-			Expect(bundles).To(ConsistOf(
-				operatorscommon.BundleVirtualization,
-				operatorscommon.BundleOpenshiftai,
-			))
+			Expect(bundles).To(HaveLen(2))
+			for _, bundle := range bundles{
+				Expect(bundle.Name).To(BeElementOf(operatorscommon.BundleVirtualization, operatorscommon.BundleOpenshiftai))
```

Comment:
As we are no longer simply processing strings but instead processing bundles, is there anything else we should expect here, I know that the Bundle also have a list of operators for example?